packages/currency/src/Balance.ts

```diff
@@ -66,6 +67,16 @@ export default class Balance<T extends BaseCurrencyType> {
     return new Balance(integerBalance, currencyType)
   }
 
+  static fromIntAndTicker(integerBalance: number, ticker?: Ticker) {
+    const currencyType = CurrencyType.fromTicker(ticker)
```

Comment:
ticker is just a string right? what if this fails to find a matching currency type? how does that get handled?